nautobot/dcim/utils.py

```diff
@@ -53,13 +54,22 @@ def cable_status_color_css(record):
     """
     Given a record such as an Interface, return the CSS needed to apply appropriate coloring to it.
     """
+    CABLE_STATUS_TO_CSS_CLASS = {}
     if not record.cable:
         return ""
-    # The status colors are for use with labels and such, and tend to be quite bright.
-    # For this function we want a much milder, mellower color suitable as a row background.
-    base_color = record.cable.get_status_color().strip("#")
-    lighter_color = rgb_to_hex(*lighten_color(*hex_to_rgb(base_color), 0.75))
-    return f"background-color: #{lighter_color}"
+    else:
+        for choice in CableStatusChoices.CHOICES:
+            color_value = STATUS_COLOR_MAP[choice[1]]
+            if color_value == ColorChoices.COLOR_GREEN:
+                CABLE_STATUS_TO_CSS_CLASS[color_value] = "success"
+            elif color_value == ColorChoices.COLOR_AMBER:
+                CABLE_STATUS_TO_CSS_CLASS[color_value] = "danger"
```

Comment:
Not `warning`? I would have thought amber would be warning and red would be danger. If this matches the current behavior though, that's fine.

Reply:
Ahh Stephen mentioned that decommisioning was red in the issue so I did not check but I double-checked it should be `warning`. Thank you for catching that.